### ISSUE TITLE
Infinite scroll fires even when we scroll a bit at top or middle?

### ISSUE THREAD

**Issue by YuvPrime** [sroze/ngInfiniteScroll#265]
Infinite scroll fires even when we scroll a bit at top or middle. 

     <div infinite-scroll='loadMore()' infinite-scroll-disabled='isbusy'>
      </div>

        $scope.isbusy = false;
        $scope.loadMore = function() {
            console.log("load");
              $scope.isbusy = true;
              // loads data
              $scope.isbusy = false;
         });

**Comment by adwtt90** [comment]
This issue is likely related to #258.  It looks like they've found a solution on that thread, but I haven't gotten to test it myself yet.

**Comment by ahmadsety** [comment]
+1

**Comment by Rahul-Giri** [comment]
+1

**Comment by AndriiDidkivsky** [comment]
+1

**Comment by graingert** [comment]
have you tried using https://material.angularjs.org/latest/demo/virtualRepeat

**Comment by AndriiDidkivsky** [comment]
@graingert 
Good idea, we are using angular-material in our project and it can resolve a problem.  I have missed this point in documentation.
